src/power_grid_model_io/converters/pandapower_converter.py

```diff
@@ -566,7 +566,35 @@ def _create_pgm_input_ward(self):  # pragma: no cover
         if pp_wards.empty:
             return
 
-        raise NotImplementedError("Ward is not implemented yet!")
+        length = len(pp_wards)
```

Comment:
I prefer variable names to be explicit, so I would call it `n_wards` or `num_wards`, instead of length.